features/step_definitions/networking.rb

```diff
@@ -744,6 +742,8 @@
     end
   end
   unless host_ovs
+    network_operator = BushSlicer::NetworkOperator.new(name: "cluster", env: env)
```

Comment:
@rbbratta probably can combine these two lines into one?

Reply:
I'd rather replace all the instances at once, but this current PR fixes current test failures, can we merge while I refactor `network_type`?